explainer/Explainer11.md

```diff
@@ -0,0 +1,287 @@
+# WoT Thing Description (TD) 1.1 Explainer (DRAFT - WIP)
+This is an updated version of the [Explainer for TD 1.0](Explainer.md).
+
+
+## What's All This About?
+
+[//]: # (What role does the TD specification play in the WoT?)
+
+[//]: # (A brief, 4-5 paragraph explanation of the feature’s value. Outline what the feature does and how it accomplishes those goals in prose. If your feature creates UI, this is a great place to show mocks and user flows.)
+
+[//]: # (The W3C Web of Things [WoT] is intended to enable
+interoperability across IoT Platforms and application domains using Web technology.)
+
+In the [WoT Architecture](https://w3c.github.io/wot-architecture/index.html), 
+a Thing is defined as an abstraction of a physical IoT device such as a sensor (temperature, CO2, ...), an actuator (lamp, motor, ...), or a service like a weather service. The Thing Description (TD) provides descriptive metadata for a Thing's interface.
```

Comment:
The definition of Thing etc. have changed, make sure they are up to date.   I think these are ok, however, since you mention both physical devices (maybe "network-connected physical devices?") and services.

I would also use "network interface" instead of just "interface" since a reader might think we are describing the "user interface", etc. (which we are not).

Reply:
done